### DISCUSSION TITLE
Can I configure this package to backup my configured s3 filesystems as well?

### DISCUSSION THREAD

**Discussion by citricguy** [spatie/laravel-backup#1467] [Q&A] [answered]
I apologize if this is common knowledge and I just missed it, but is there a way to get this package to take a snapshot or some kind of recoverable point-in-time backup of my configured AWS S3 or DigitalOcean Spaces filesystems?

**Comment by erikn69** [comment, marked as answer]
https://github.com/spatie/laravel-backup/issues/1431#issuecomment-1014955615

**Reply by citricguy** [reply]
Thank you.
